python/pyproject.toml

```diff
@@ -1,3 +1,15 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+[build-system]
+
+requires = [
+    "wheel",
+    "setuptools",
+    "numba>=0.49",
+    "Cython>=0.29,<0.30",
+    "numpy",
```

Comment:
I think this is only for defining build tools and not general install requirements.

Reply:
Added the required packages into `install_requires` in setup.cfg to be independent from build req.

Reply:
I don't think we need `numba` or `numpy` in this section as this is just the build system.

Reply:
Removed here.. 👍